kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -547,7 +547,7 @@ servicesComponentFactorySelectorIdle=--- Select Component Factory---
 servicesComponentFactoryNew=New Component
 servicesComponentFactoryFactory=Factory
 servicesComponentFactoryName=Name
-servicesComponentFactoryNamePlaceholder=Please enter an user friendly name
+servicesComponentFactoryNamePlaceholder=Please enter a user friendly name
 servicesComponentFactoryAlertNotSelected=Component must be selected and the name must be non-empty
```

Comment:
Shouldn't it make more sense if we write as "the name must not be empty"?. Just thinking.

Reply:
This isn't a validation message, but a text placeholder. So I think the text which is there is more appropriate.